.circleci/src/commands/notify.yml

```diff
@@ -0,0 +1,37 @@
+description: Send a notification (to Mattermost) at the end of a job, based on success or failure. Must be the last step in a job.
+parameters:
+  success_message:
+    default: ''
+    type: string
+  failure_mentions:
+    default: '@stefanie.metzger @andreas.fischer'
+    type: string
+  success_mentions:
+    default: ''
+    type: string
+  allow-all-branches:
+    default: false
+    type: boolean
+  channel:
+    default: entitlementcard-notifications
+    type: string
+steps:
+  - run:
+      command: |
+        echo 'export MM_MESSAGE=":fire: The [${CIRCLE_JOB}](${CIRCLE_BUILD_URL}) job has failed on the ${CIRCLE_BRANCH} branch! :fire: << parameters.failure_mentions >>"' >> $BASH_ENV
+      name: Mattermost - Prepare failure message
+      when: on_fail
+  - run:
+      command: |
+        if [ -n "<< parameters.success_message >>" ]; then
+            echo 'export MM_MESSAGE="<< parameters.success_message >>"' >> $BASH_ENV
+        fi
+      name: Mattermost - Prepare success message
+      when: on_success
+  - run:
+      command: |
+        echo 'export ALLOW_ALL_BRANCHES="<< parameters.allow-all-branches >>"' >> $BASH_ENV
+        app-toolbelt v0 notify mattermost --message "${MM_MESSAGE}" --channel << parameters.channel >> ${ALLOW_ALL_BRANCHES:+--allow-all-branches}
```

Comment:
What's this syntax/What does the '+' mean? `${ALLOW_ALL_BRANCHES:+--allow-all-branches}`

Reply:
It is the bash syntax for conditionally set the `allow-all-branches`-flag when the ALLOW_ALL_BRANCHES env var is set.